docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * Update documentation formatting for ReadTheDocs.io Markdown requirements.
+  * Restructure some file layout for ReadTheDocs.io build system, including: `CHANGELOG`, `LICENSE`, `CONTRIBUTING`, `MIGRATION_GUIDE`, `PERFORMANCE`, `SEMANTIC-VERSIONING`, and `TEST` files.
+  * Corrected order of `low_emission_zone` in list.
+  * Generalized `hgv_restriction` property units to mostly meters for heavy goods vehicle truck access restrictions.
+
+  #### INTERNAL CHANGES
+
+  * Add `gunicorn` to dependencies, thanks [@rwrx](https://github.com/rwrx). [PR #1690](https://github.com/tilezen/vector-datasource/pull/1690)
+  * Bump `PyYAML` version for CVE-2017-18342.
+  * Stop using `tags-> in YAML (Issue [#1199](https://github.com/tilezen/vector-datasource/issues/1199))
+  * Add ability to make tests from relations, and calculate the area for TODO
+  * Estimate `capacity` for parking lots and garages based on area and parking type with new `tz_estimate_parking_capacity` function.
+  * Protect against `None` shapes in way area calculation.
+  * Add `all_the_kinds` script to output all the `kind` and `kind_detail` values per `$layer`, with their `min_zoom` from the YAML files.
+  * Add whitelists to `kind_detail` in each layer YAML so that we can enumerate all possible values for the script.
+  * Extend Natural Earth test generator to support polygons.
+  * Use new `drop_names` post-process function to drop all the names, not just the TODO
+  * Add `CollisionRanker` to support YAML-based spreadsheets for `collision_rank` logic, including reserved, gaps, and filters across layers.
+  * Add `safe_int` implementation for SQL.
+  * Expose a test method that can return the whole tile. Used in new `collision_rank` tests.
+  * Clip to Mercator world bounds before projecting shapefiles.
+  * Deal with 'download only' test mode.
+  * Skip download only tests after downloading fixtures (if any).
+  TODO osm land/water
+  * Update assets bundle for #1552 & #1809, and updated OSMdata.com land, water.
+  * Reorder under specific national parks, but above the generic tier2 na…  …
+  * Guard against future airport runway polygons in landuse versus runway lines in roads `sort_rank` errors with a test.
+  * Don't create a zero area property on points in SQL templating.
+  * Support shapefiles in ZIP for asset bundle creation.
+  * Add support for `==` operator in YAML evaluation.
+  * Add new `drop_names` function to drop all name variant properties.
+  * Add new `remap` function to remap landuse kind values at low- and mid-zooms.
+  * Add new `add_vehicle_restrictions` function for hgv.
+  * Add new `add_collision_rank` function.
+  * Add new `update_min_zoom` function.
+  * Refactor order of landuse layer property dropping, and small inner geometry dropping for merging.
+  * Move Natural Earth feature class to kind mapping to Python in `remap_viewpoint_kinds`.
+  * Add new function to support remapping Natural Earth's points of view to kinds and drop null values.
```

Comment:
Isn't this `remap`, as described above?

Reply:
The first remap is generic function, and the POV stuff is 2nd function. Added the name but moved the two lines to be next to each other in 64520d8e5d5866667a755d898e5561361a742d1c.